app/jobs/data_export_task_job.rb

```diff
@@ -2,11 +2,11 @@ class DataExportTaskJob < TaskJob
   def perform(job_params)
     data_export = DataExport.find(job_params[:data_export_id])
     begin
+      task_for_job(self).set_activity_message("data_export_in_progress")
       ::DataExportService.run(data_export)
     rescue DataExportError => e
       task_for_job(self).update(custom_error_data: e.child_errors)
       task_for_job(self).set_activity_message("finished_with_custom_error_data")
-      raise e
```

Comment:
raising this error makes the task appear to have failed, when in fact the csv export is available.

Reply:
I'm still not sure if that's desirable behavior or not. Should we create a new status for tasks, something like "succeeded with errors" or "partially succeeded"? I want to make sure we also don't call exports that have problems unambiguous successes.